### PR TITLE
Add news item for Circulars over Kafka and schema v4.1.0

### LINKED ISSUE #2343: Add announcement for GCN Circulars over Kafka and schema release
# Description

Add a news item for GCN Circulars over Kafka, which are now live.

# Acceptance criteria

- Add an item to the news page.
- Update the news banner.
- schema release: EP adding ID, circulars, pointing core schema

### PR DESCRIPTION
**Pull request by jracusin**
# Description
This PR provides a news item for Circulars over Kafka and schema v4.1.0.

# Related Issue(s)
Resolves #2343

### CODE REVIEW

**Review by lpsinger — COMMENTED** [review]
I suggest that we wait until we have merged https://github.com/nasa-gcn/gcn-schema/pull/154 to announce this.

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:39]: - Eliminate passive voice.
- Don't refer to it as the "QuickStart interface"; we never use the term "QuickStart" except in the URL.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Can we also advertise the new heartbeat topic?

File: app/root.tsx
Comment (lpsinger) [inline app/root.tsx:253]: How about:

```suggestion
          New! Circulars over Kafka, heartbeat topic, and schema v4.1.0. See{' '}
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:43]:
```suggestion
          - Modification in the [Pointing core schema](https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices/core/Pointing.schema.json) from `ra` and `dec` to `ra_pointing` and `dec_pointing` to remove ambiguity with source position
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:41]: Rather than "Inclusion of" and "Modification of" how about "Include" and "Modify" so that these resemble complete sentences in the imperative mood?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:42]:
```suggestion
          - `id` property (from the Event core schema) in [Einstein Probe Notices](https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices/einstein_probe/wxt/alert.schema.json)
```

**Review by lpsinger — CHANGES_REQUESTED** [review]
Be consistent about whether the bullet points end with punctuation.

**Review by lpsinger — CHANGES_REQUESTED** [review]
As I said before:

> Be consistent about whether the bullet points end with punctuation.

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:36]: - Reorder so that the order of this list agrees with the news banner.
- Always use the Oxford comma.

```suggestion
        <Anchor>Circulars are Now Available via Kafka, Heartbeat Kafka Topic, and New Schema Release v4.1.0</Anchor>
```

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx:39]:
```suggestion
        - **Stream Circulars over Kafka in JSON format** on the topic `gcn.circulars`.  Add Circulars to your existing Kafka streaming codes or generated via the [Start Streaming GCN Notices interface](/quickstart).
```

File: app/routes/news._index.mdx
Comment (lpsinger) [inline app/routes/news._index.mdx]: Reorder these bullet points to match the order in the news banner.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/news._index.mdx
Comment (jracusin) [reply app/routes/news._index.mdx:39]: It's not just notices anymore.

Comment (lpsinger) [reply app/routes/news._index.mdx:39]: That's true, but it's still what that interface is called. The naming is inconsistent now, but that's what it's called.

**Review by lpsinger — APPROVED** [review]

**Comment by lpsinger** [comment]
When you merge a PR on which you made multiple fixup commits, _please squash your changes_.

**Pull request merged**
